.github/workflows/ci.yml

```diff
@@ -48,4 +48,6 @@ jobs:
         if: failure() && steps.playwright.outcome == 'failure'
         with:
           name: artifacts-${{ matrix.os }}
-          path: run/playwright-itest
+          path: |
+            run/playwright-itest
+            !run/**/SS
```

Comment:
This got our artifact uploads working again. We ignore the symlinked files that electron created.